apps/dev-recruiters/src/components/modules/OpportunitiesAggregatorWithRoles/OpportunitiesAggregator.tsx

```diff
@@ -4,13 +4,15 @@ import {
 
 import * as React from "react";
 import Slider from "react-slick";
-import "slick-carousel/slick/slick-theme.css";
+// uncomment the below line at your own risk, it has broken /join page
+//import "slick-carousel/slick/slick-theme.css";
 import "slick-carousel/slick/slick.css";
 import BoxContainer from "../../common/BoxContainer";
 import FilteringComponent, {
   FilteringComponentProps
 } from "./filtering/FilteringComponent";
 import ShortCard from "./ShortCard";
+import '../hide-slick-dots.css';
```

Comment:
@dbradham 
I think commenting the import statement of `slick-carousel/slick/slick-theme.css` has nothing to do with the app breaking on /join. can please uncomment it and remove line `import '../hide-slick-dots.css';` and its file

Reply:
If I uncomment that line, I see the below errors and the application becomes unresponsive as soon as I click /join. I'm open to other suggestions to solve this, as this is not the most beautiful solution. (but is a needed solution from my testing)

<img width="755" alt="Screen Shot 2023-02-20 at 11 19 55 AM" src="https://user-images.githubusercontent.com/43216378/220168603-d5b0deec-9235-4355-b6a1-58d047e18d3e.png">

Reply:
@dbradham that's probably an issue with your environment, mind if you send me node version and the os?
on my side everything works as expected without commenting the slick css and it's important for the functionality of the carousel

Reply:
for me I'm using 
node: v14.17.0
os: Pop!_OS 22.04 LTS

Reply:
@Enjoy2Live Thanks that seems to be the issue. I have been using node v18.12.1 on Monterey OS (12.3). I tried on my windows 11 PC with v16.16.0 Node and it seems to be working as expected. I just updated the PR to revert these changes.